Fix various styling issues in `Play_style` (#7077)
ppy/osu-wiki: wiki/Play_style/en.md

### Mouse-only

> Mouse-only was once the *only* way to play osu! until other playing methods were supported over time.

Author: This sentence is used in similar ways throughout the article, but every other version doesn't have a note.

> Mouse-only was once the *only* way to play osu! until other playing methods were supported over time.
> 
> Also known as *pure-mouse*, mouse-only players navigate the in-game cursor with a mouse while tapping beats, following sliders, and spinning spinners with only clicking the left and right mouse buttons.

Reviewer: ```suggestion
Also known as *pure-mouse*, mouse-only players navigate the in-game cursor with a mouse while tapping beats, following sliders, and spinning spinners with only clicking the left and right mouse buttons.

Mouse-only used to be the *only* way to play osu! until other playing methods were supported over time.
```

### Taiko controller

A taiko controller is the default play style and controller for the [Taiko no Tatsujin](https://wikipedia.org/wiki/Taiko_no_Tatsujin) series ([Taiko Drum Master](https://wikipedia.org/wiki/Taiko:_Drum_Master) for US).

*Note: Some skilled players who play using a Taiko controller may create theirs from scratch while others purchase theirs via retail or online stores.*

Players who use a Taiko controller carry a Taiko/drum stick in each hand and hit the centre (Red) or rim of the drum (Blue) to register the note based on where it is hit.

> *Notice: Most Taiko controllers connect to the computer by using a Bluetooth connector should there be no USB connector. Though this may increase latency.*

Reviewer: ```suggestion
*Notice: Most Taiko controllers connect to the computer by using a Bluetooth connector should there be no USB connector, though this may increase latency.*
```

### Keyboard

> *Note: For more information about mechanical keyboards, see this [external guide for mechanical keyboards](http://www.daskeyboard.com/blog/mechanical-keyboard-guide/).*

Author: I thought that this is better, as to having half-floating links in the article.

Keyboard was once the *only* way to play osu!mania until other play methods was supported over time.

> To play with a keyboard, players would press down the keys respective to their placement on the in-game keyboard. By default these keys are `D` `F` `J` `K`. The advantage of this play-style is that it can accommodate to basically all the keys settings.

Reviewer: ```suggestion
To play with a keyboard, players would press down the keys respective to their placement on the in-game keyboard. By default these keys are `D`, `F`, `J`, and `K`. The advantage of this play style is that it can accommodate to basically all the keys settings.
```

Author: There is a similar listing of keys in line 94

> *Note: For key-binding, the default is `Z` `X` `C` `V` (KDDK, or BRRB).*

Should this also be changed like this here?

Reviewer: yeah i suppose

### Other styles

> - **Guitar/Drum set** (5/6K): The default controller for the guitar-themed ([Guitar Hero](https://wikipedia.org/wiki/Guitar_Hero))/drum-themed ([Drummania](https://wikipedia.org/wiki/Drum_Mania)) rhythm game. Players stream on their guitar/hit the drum or cymbals to input commands. Drum players have a disadvantage compared to guitar players because they can only input two commands at a time without a special kick pedal.

Reviewer: ```suggestion
- **Guitar/Drum set** (5/6K): The default controller for the guitar-themed ([Guitar Hero](https://wikipedia.org/wiki/Guitar_Hero))/drum-themed ([DrumMania](https://en.wikipedia.org/wiki/GuitarFreaks_and_DrumMania)) rhythm game. Players stream on their guitar or hit the drums and cymbals to input commands. Drum players have a disadvantage compared to guitar players because they can only input two commands at a time without a special kick pedal.
```
